kpi/tests/api/v2/test_api_paired_data.py

```diff
@@ -391,21 +391,13 @@ def test_get_external_with_no_auth(self):
         # When owner's destination asset does not require any authentications,
         # everybody can see their data
         self.client.logout()
-        response = self.client.get(self.external_xml_url)
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
-
-    def test_get_external_from_owner_with_extra_detail(self):
-        self.deploy_source()
-        # When owner's destination asset does not require any authentications,
-        # everybody can see their data
-        self.client.logout()
-
-        # Remove owner's extra detail
-        ExtraUserDetail.objects.filter(user=self.anotheruser).delete()
-        self.anotheruser.refresh_from_db()
-
-        response = self.client.get(self.external_xml_url)
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        with patch(
+            'kpi.deployment_backends.backends.MockDeploymentBackend.xform',
+            MagicMock(),
+        ) as xf_mock:
+            type(xf_mock).require_auth = PropertyMock(return_value=False)
```

Comment:
Unfortunately, you won't learn anything today. It's a wrong try/error mock of a non-existing property (i.e.: `MockDeploymentBackend.xform`) but I could not make it work so I added a dummy property `xform` to `MockDeploymentBackend`. 
Long story short: this can be simplified with `xf_mock.require_auth = False`